### ISSUE TITLE
Pool{Type}Array has no method 'has'

### ISSUE THREAD

**Issue by toby3d** [godotengine/godot#41075]
**Godot version:** 3.2.3.rc1
**OS/device including version:** Linux 5.7.9-1-MANJARO
**Issue description:** Unlike `Array`, `PoolStringArray`, `PoolIntArray` and others doesn't have a method to check if an element is present.

**Steps to reproduce / Minimal reproduction project:**
```gdscript
var _array: Array = ['a', 'b', 'c']
_array.has('b') # true
_array.has('z') # false
```
```gdscript
var _array: PoolStringArray = ['a', 'b', 'c']
_array.has('b') # The method "has" isn't declared on base "PoolStringArray"
```

There are two workarounds to solve this:
```gdscript
var _array: PoolStringArray = ['a', 'b', 'c']
(_array as Array).has('b') # true
(_array as Array).has('z') # false
```

```gdscript
static func has_string(src: PoolStringArray, find: String) -> bool:
	for item in src:
		if item != find:
			continue
		
		return true

	return false

...

var _array: PoolStringArray = ['a', 'b', 'c']
has_string(_array, 'b') # true
has_string(_array, 'z') # false
```

**Comment by Calinou** [comment]
This was implemented by https://github.com/godotengine/godot/pull/32144, but I'm not sure if it's worth remaking for the `3.2` branch.

**Comment by timothyqiu** [comment]
Fixed by #60856
